conda.recipe/meta.yaml

```diff
@@ -4,6 +4,8 @@ package:
 
 build:
   number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
+  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
```

Comment:
Yeah, I see the value of the slightly more complex solution.  Again, I'm fine with this change, particularly since you seem to be managing most of this stuff these days.